sdk/rust/Cargo.toml

```diff
@@ -15,7 +15,7 @@
 
 [package]
 name = "sawtooth-sdk"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
Is this for the changed consensus behavior? (expecting activation message from validator)

Reply:
Yep!